docs/source/cross_validation.rst

```diff
@@ -36,4 +35,48 @@ However, if there's a strong pattern in your data, you'll want to perform a full
 Second, the Dask version avoids allocating large intermediate NumPy arrays storing the indexes for slicing.
 For very large datasets, creating and transmitting ``np.arange(n_samples)`` can be expensive.
 
+Here is another illustration of performing k-fold cross validation purely in Dask. Here a link to gather more information on k-fold cross validation :class:`dask_ml.model_selection.KFold`:
+
+.. ipython:: python
+
+   import dask.array as da
+   from dask_ml.model_selection import KFold
+   from dask_ml.datasets import make_regression
+   from dask_ml.linear_model import LinearRegression
+   from statistics import mean 
+
+   X, y = make_regression(n_samples=200, # choosing number of observations
+				 n_features=5, # number of features
+				 random_state=0, # random seed
+				 chunks=20) # partitions to be made 
+
+   train_scores: list[int] = []
+   test_scores: list[int] = []
+
+   model = LinearRegression()
+
+The Dask kFold method splits the data into k consecutive subsets of data. Here we specify k to be 5, hence, 5-fold cross validation
+
+
+.. ipython:: python
+   kf = KFold(n_splits=5)
+
+   for i, j in kf.split(X):
```

Comment:
Looks like the doc build failed here: https://github.com/dask/dask-ml/actions/runs/9434455032/job/26270343032?pr=994

Is that from the lack of a newline after the `ipython` directive?

Reply:
thanks you for spotting that, added a new line